Package.swift

```diff
@@ -9,14 +9,27 @@ let package = Package(
         .library(
             name: "Apollo",
             targets: ["Apollo"]),
+		.library(
+			name: "ApolloSQLite",
+			targets: ["ApolloSQLite"]),
+		.library(
+			name: "ApolloWebSocket",
+			targets: ["ApolloWebSocket"]),
     ],
     dependencies: [
-
+		.package(url: "https://github.com/stephencelis/SQLite.swift.git", from: .init(0, 12, 2)),
+		.package(url: "https://github.com/daltoniam/Starscream", from: .init(3, 1, 0)),
```

Comment:
IIRC you can do these as `"3.1.0"` string literals - I kinda prefer that from a readability standpoint but I can also fix that myself if you'd prefer

Reply:
Ah no problem; would you prefer `.upToNextMajor` or `.exact`? (Or did you mean _just_ the literal? I couldn't get that to work in Xcode 11b4…)

Reply:
`exact` would be lovely - it's possible they yanked the ability to just use a string literal.